app/views/tagged_content/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= render "govuk_component/title", title: @taxon.title %>
+
+<div class="list-page">
+  <h2 class="list-page__title"><%= @tagged_content.document_type.humanize %> Items - <%= @tagged_content.search_results_count %></h2>
+  <div>
+    <ul class="list-page__content_list">
```

Comment:
Following BEM, this should be `list-page__content-list` (dash between content and list, rather than underscore)

Reply:
Ah yeah! Thanks for seeing that.